### PR TITLE
DBus tests for Filesystems.BTRFS

### PR DESCRIPTION
**Pull request by vojtechtrefny**

### CODE REVIEW

File: src/tests/dbus-tests/test_btrfs.py [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+        # remove the second device
+        with self._temp_mount(dev1.path):
+            dev1.obj.RemoveDevice(dev2.path, self.no_options,
+                                  dbus_interface=self.iface_prefix + '.Filesystem.BTRFS')
+            fstype = self.get_property(dev2.obj, '.Block', 'IdType')
+            fstype.assertFalse()
+
+        # check number of devices
+        # storaged currently reports wrong number of devices (2)
```
Comment (vpodzime) [inline src/tests/dbus-tests/test_btrfs.py]: Please mark this comment with ``XXX:`` or ``BUG:`` so that we can easily find it in the future.

Comment (vojtechtrefny) [inline src/tests/dbus-tests/test_btrfs.py]: Fixed

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Comment by vpodzime** [comment]
Jenkins, test this, please.

**Comment by vpodzime** [comment]
Jenkins, test this, please.

**Pull request merged**